kuka_rsi_hw_interface/krl/KR_C4/README.md

```diff
@@ -100,21 +109,20 @@ $ roslaunch kuka_rsi_hw_interface test_hardware_interface.launch sim:=false
 
 The **kuka_rsi_hardware_interface** is now waiting for the robot controller to connect. Follow the next steps to connect RSI:
 
-1. On the teach pad, enter mode **T1** for testing purposes.
-2. Navigate to `KRC:\R1\Program` and select `ros_rsi.src`.
-3. Press and hold an enabling switch and the run/play-button. The robot will first move to the start position.
+1. Navigate to `KRC:\R1\Program` and select `ros_rsi.src`.
+2. Press and hold an enabling switch and the run/play-button. The robot will first move to the start position.
    * A message like **Programmed path reached (BCO)** will be shown at the teach pad.
-4. Press and hold again. The teach pad will post a warning **!!! Attention - Sensor correction goes active !!!**.
-5. Confirm the warning and press and hold the buttons again. This time the terminal where **kuka_rsi_hardware_interface** is running should output **Got connection from robot**. The RSI connection is now up and running.
-6. In a new terminal:
+3. Press and hold again. The teach pad will post a warning **!!! Attention - Sensor correction goes active !!!**.
+4. Confirm the warning and press and hold the buttons again. This time the terminal where **kuka_rsi_hardware_interface** is running should output **Got connection from robot**. The RSI connection is now up and running.
+5. In a new terminal:
 
 ```
 $ rosrun rqt_joint_trajectory_controller rqt_joint_trajectory_controller
 ```
 
 Choose **controller manager ns** and **controller** and you should be able to move each robot joint.
 
-* Note that T1-mode limits the robot movement velocity and is intended for testing purposes.
+* Note that T1-mode is recommended as it limits the robot movement velocity and is intended for testing purposes.

```

Comment:
Is that true? Does it work? Do you need to keep the green play button on the robot panel pressed?